source/content/guides/drupal-9/drupal-9-hosted-createempty-md/05-composer-upstream.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Migrate a Site That Was Created with an Empty Upstream to Drupal 9
+subtitle: Add the Integrated Composer Upstream in a New Local Branch
+description: 
+cms: "Drupal 9"
+categories: [develop]
+tags: [code, launch, migrate, site, updates]
+contributors: [wordsmither]
+reviewed: "2021-03-31"
+layout: guide
+showtoc: true
+permalink: docs/guides/drupal-9-hosted-createempty-md/composer-upstream
+anchorid: composer-upstream
+editpath: drupal-9/drupal-9-hosted-createempty-md/05-composer-upstream.md
+---
+This process involves significant changes to the codebase that may take some time to complete, and can be complicated to roll back. 
+
+To minimize issues, these steps make the codebase changes in a new branch:
+
+1. In your local terminal, change directories to the site project. For example, if you keep your projects in a folder called `projects` in the home directory:
+
+  ```bash{promptUser:user}
+  cd ~/projects/$SITE/
+  ```
+
+1. Add the Pantheon Drupal Project upstream as a new remote called `ic`, fetch the `ic` upstream, and checkout to a new local branch based on it called `composerify`:
+
+  ```bash{outputLines:2}
+  git remote add ic git@github.com:pantheon-upstreams/drupal-recommended.git && git fetch ic && git checkout --no-track -b composerify ic/master
```

Comment:
This could be changed to drupal-composer-managed right away to avoid changing it in the near future